cross/lua/patches/001_Makefile_add_shared_library.patch

```diff
@@ -0,0 +1,45 @@
+--- Makefile	2017-04-26 18:12:27.442079208 +0200
```

Comment:
Please add a comment with explanation what these patches are used for. And if coming from an external source, URL to its source.